packages/near-api-js/test/account.test.js

```diff
@@ -113,15 +117,18 @@ describe('with deploy contract', () => {
     });
 
     beforeEach(async () => {
-        oldLog = console.log;
         logs = [];
-        console.log = function () {
-            logs.push(Array.from(arguments).join(' '));
-        };
+        configureLogging({
+            showLogs: () => true,
+            onLog: (args) => {
+                logs.push(Array.from(args).join(' '));
+            },
+            onWarn: () => {}, // Do nothing onWarn
+        });
```

Comment:
Same here, need to spread `args` in the `onLog` property to recreate old behavior